src/build.sh

```diff
@@ -313,15 +317,19 @@ for branch in ${BRANCH_NAME//,/ }; do
         # Start the build
         echo ">> [$(date)] Starting build for $codename, $branch branch" | tee -a "$DEBUG_LOG"
         build_successful=false
+        set +eu
+        # shellcheck source=/dev/null
         if mka bacon &>> "$DEBUG_LOG"; then
+          set -eu
+
```

Comment:
Is this change needed? I thought `mka` was a program and not a function, in which case it needs neither this nor the subshell wrapping (i.e. no `set +eu` at all).

In any case, `# shellcheck source=/dev/null` is also unnecessary here, as it does nothing to commands which are not `source`.

Reply:
`mka` is actually another shell function: https://github.com/LineageOS/android_vendor_lineage/blob/7124e2cdbc66739ad80b160d222426ec5f9c2835/build/envsetup.sh#L719

running it without `set +e` failed for me, but running it in a subshell with it worked and seems like a better solution